sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2067,6 +2067,18 @@ object functions {
    */
   def radians(columnName: String): Column = radians(Column(columnName))
 
+  /**
+   * returns number truncated by specified decimal places.
+   *
+   * @param scale: 4. -4, 0
+   *
+   * @group math_funcs
+   * @since 2.3.0
+   */
+  def trunc(db: Column, scale: Int = 0): Column = withExpr {
```

Comment:
I would avoid using default value here unless we are very sure on this (e.g., other languages can call this without any problem, this guarantees method signature and does not break binary compatibility and etc.).